src/controller/BackPostController.php

```diff
@@ -67,7 +67,7 @@ public function addPost(PostModel $postModel)
 		];
 
 		// display the form with errors and datas form
-		$this->render('insert_post.twig', $errors, $form, $_SESSION);
+		$this->render('insert_post.twig', $errors, $form, $_SESSION, []);
```

Comment:
A modifier si tu changes ta fonction `render()`

Reply:
Modif OK : 
$this->render('insert_post.twig', $_SESSION, $errors, $form);